doc/source/ray-core/tasks/resources.rst

```diff
@@ -13,12 +13,11 @@ available to execute the task.
 
 .. tabbed:: Python
 
-    .. code-block:: python
```

Comment:
You can skip resources page for now since I'm moving it to `scheduling/resources`